python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -18,158 +18,238 @@ Available constructors:
-easily be deduced from the coefficients :math:`\alpha_k`.
+- definition of the multivariate orthonormal basis;
+- truncation of the multivariate orthonormal basis;
+- evaluation of the coefficients.
 
-We detail here all the steps required in order to create a functional chaos 
-algorithm.
+These steps are presented in more detail below.
 
-**Step 1 - Construction of the multivariate orthonormal basis**: the
+**Step 1 - Definition of the multivariate orthonormal basis**: the
 multivariate orthonornal basis :math:`(\Psi_k(\vect{x}))_{k \in \Nset}` is built
 as the tensor product of orthonormal univariate families.
 
 The univariate bases may be:
 
-    - *polynomials*: the associated distribution :math:`\mu_i` is continuous or discrete. 
-      Note that it is possible to build the polynomial family orthonormal to any univariate 
-      distribution :math:`\mu_i` under some conditions. 
-      For more details, see :class:`~openturns.StandardDistributionPolynomialFactory`;
+- *polynomials*: the associated distribution :math:`\mu_i` can be continuous
+  or discrete.
+  Note that it is possible to build the polynomial family orthonormal to any
+  arbitrary univariate distribution :math:`\mu_i` under some conditions. 
+  For more details on this basis, see :class:`~openturns.StandardDistributionPolynomialFactory`;
 
-    - Haar wavelets: they enable to approximate functions with discontinuities.
-      For more details, see :class:`~openturns.HaarWaveletFactory`,;
+- Haar wavelets: they enable to approximate functions with discontinuities.
+  For more details on this basis, see :class:`~openturns.HaarWaveletFactory`;
 
-    - Fourier series: for more details, see :class:`~openturns.FourierSeriesFactory`.
+- Fourier series: for more details on this basis, see :class:`~openturns.FourierSeriesFactory`.
 
-Furthermore, the numerotation of the multivariate orthonormal basis :math:`(\Psi_k(\vect{z}))_k` 
-is given by an enumerate function which defines a regular way to generate the collection of degres 
-used for the univariate polynomials : an enumerate function represents a bijection 
-:math:`\Nset \rightarrow \Nset^d`. See :class:`~openturns.LinearEnumerateFunction` or :class:`~openturns.HyperbolicAnisotropicEnumerateFunction` 
-for more details.
+Furthermore, the numbering of the multivariate orthonormal basis
+:math:`(\Psi_k(\vect{z}))_{k \in \Nset}` is given by an enumerate function
+which defines a way to generate the collection of polynomial degrees used
+for the univariate polynomials: an enumerate function
+represents a bijection :math:`\Nset \rightarrow \Nset^{n_X}`.
+See :class:`~openturns.LinearEnumerateFunction` or
+:class:`~openturns.HyperbolicAnisotropicEnumerateFunction` for more details
+on this topic.
 
-**Step 2 - Truncation strategy of the multivariate orthonormal basis**: a
+**Step 2 - Truncation of the multivariate orthonormal basis**: a
 strategy must be chosen for the selection of the different terms of the
```

Comment:
Maybe mention it is the adaptive strategy. I can't make the suggestion, Github won't let me.

Reply:
The adaptive strategy is usually considered as a method to compute the coefficients. This is why the current text seems to be correct to me.

Reply:
OK